### DISCUSSION TITLE
How to test ui component locally

### DISCUSSION THREAD

**Discussion by Sahillather002** [yamada-ui/yamada-ui#845] [General]
I was trying to test ui component say feature but couldn't able to get the idea behind it .

![image](https://github.com/hirotomoyamada/yamada-ui/assets/72077931/78e16dcd-9a8d-4b1a-9d8a-ae011926ffe8)

I was thinking of installing the library then test it but surely it shouldn't be tested like that .
Please help me how to that.
@hirotomoyamada

**Comment by hirotomoyamada** [comment]
Hi, thanks for the comment!

> I was thinking of installing the library then test it but surely it shouldn't be tested like that .

Could you please explain thsi sentence a little more specifically?

**Reply by Sahillather002** [reply]
I was just finding way to test the component nothing special this line .

**Reply by setodeve** [reply]
@Sahillather002 
you can run the test only for `breadcrumb` by executing `pnpm test breadcrumb` on yamada-ui root directory.
